### ISSUE TITLE
System test pod creation fails for multiple custom Pravega config parameters.

### ISSUE THREAD

**Issue by ShwethaSNayak** [pravega/pravega#6723]
**Problem description**
If user passes multiple custom config parameters then system test pod creation fails with below error

```
2022-04-01T17:20:37.945839988Z stderr F Usage: java [options] <mainclass> [args...]
2022-04-01T17:20:37.94588221Z stderr F 

2022-04-01T17:20:37.997725529Z stderr F /bin/sh: 2: storage.appends.enable=false: not found
```

**Comment by ShwethaSNayak** [comment]
```
pravegaservice.storage.layout=CHUNKED_STORAGE
storage.appends.enable=false
storage.readindex.segments.max=2048
```

**Comment by RaulGracia** [comment]
Closed via #6722.
